src/docs/sdk/features.mdx

```diff
@@ -268,6 +268,12 @@ If Performance Monitoring is both supported by the SDK and enabled in the client
 - span status must match HTTP response status code ([see Span status to HTTP status code mapping](/sdk/event-payloads/span/))
 - when network error occurs, span status must be set to `internal_error`
 
+The `url` in breadcumbs and span descriptions must be stripped of data worth protecting. Query parameters and username/password must be removed.
```

Comment:
```suggestion
If an `URL` is used in breadcrumbs or as a span description, it may contain sensitive data, such as query parameters or username/password, that must be removed by default.
```